tools/extra/hssi/ethernet/hssimac.py

```diff
@@ -129,12 +152,12 @@ def main():
         sys.exit(1)
 
     args.hssi_grps = f.find_hssi_group(devs[0].get('pcie_address'))
-    print("args.hssi_grps", args.hssi_grps)
+    print("args.hssi_grps".format(args.hssi_grps))
```

Comment:
Needs a format specifier in the string part {}

Reply:
fixed